hangforever/hangzone-frontend
# Add react router

## Description

Adds react router using this tutorial
https://www.robinwieruch.de/complete-firebase-authentication-react-tutorial#react-router-for-firebase-auth

closes #12 

## How to check

- `git fetch origin feature/add-router`
- `git checkout feature/add-router`
- `yarn dev`
- see if you can add a route
- see if you can add a link to that route
